### ISSUE TITLE
Orderbook and Trades History on Buy and Sell Page

### ISSUE THREAD

**Issue by RonaldDuck** [tomochain/bug-reports#11]
I would like to propose a new feature, it is not that big but i think it will make the trading easier and more convenient. As you can see, if i press the buy or sell button, it will go to the buy and sell page, but there is a wide space below. I think it would be easier if you also put the orderbook and trades history in that space. In that way, traders will not exit again the buy and sell page if they wanted to see the orderbook before doing a trade. I hope this will be a little help for the TomoX.
![6481458B-6998-4622-A64C-7AE902FE24F6](https://user-images.githubusercontent.com/58533399/71650631-305a1680-2d52-11ea-811b-08de34ec4bac.jpeg)
![Uploading 5342D293-7D14-4785-A1B0-10CA6E7CAD80.jpeg…]()

**Comment by thanhson1085** [comment]
I agree this suggestion. Actually, my team already discussed about this improvement, and plan to add this feature in the next phase.
Let's us know your wallet address, we will process the bounty. @chinhnt23
I also move this issue to TomoXSDK repo to add it into the plan.
